troupon/static/scss/partials/_components.scss

```diff
@@ -420,4 +424,75 @@ Other components.
       color: color(accent);
     }
   }
+}
+
+.jumbotron {
+  & > * {
+    margin-bottom: 15px;
+  }
+  .title {
+    color: color(h1);
+    font-size: 70px;
+    letter-spacing: -2px;
+  }
+}
+
+.breadcrumbs {
+  @include unstyled-h-list-mixin();
+  right: 10px;
+  
+  li.active {
+    color: color(accent);
+  }
+  li.delim {
+    font-size: font-size(breadcrumbs-caret);
+    color: color(icon-light);
+    vertical-align: middle;
+  }
+}
+
+.sidebar {
+  @include unstyled-list-mixin();
+  border-right: 1px solid color(divider);
+  li {
+    position: relative;
+    min-height: component-height(sidebar-item);
+    margin-right: -1px;
+    a, span {
+      @include v-middle;
+    }
+  }
+  li.active {
+    color: color(accent);
+    border-right: 1px solid color(accent);
+  }
+}
+
+.flash-message {
+  @include shaded-border;
+  position: relative;
+  min-height: component-height(flash-message);
+  margin-bottom: 5px;
+
+  & > [class*="col-"] {
+    min-height: component-height(flash-message);
+  }
+
+  .message-line {
+    position: absolute;

```

Comment:
Properties should be ordered color, display, font-weight, left, position, top, transform, width